target_postgres/__init__.py

```diff
@@ -40,6 +38,6 @@ def main(config, input_stream=None):
 
 
 def cli():
-    args = utils.parse_args(REQUIRED_CONFIG_KEYS)
```

Comment:
So I think we're going to want something akin to this as the failure mode for folks trying to run this when they _don't_ have the proper values present. Maybe a simple helper function which mimics the functionality of `parse_args` but does so with the fallbacks of the `environ`?